lib/compiler.js

```diff
@@ -71,6 +71,10 @@ module.exports = function compile(options) {
 
   bootDirs.forEach(function(dir) {
     bootScripts = bootScripts.concat(findScripts(dir));
+    if (env) {
```

Comment:
But I'm unsure if we should check this existed variable, because the `env` still is able to be set to `null` or `undefined` just like:

```js
boot.compile({env: null})
```

Any thoughts?